qemu/tests/cfg/win_video_play.cfg

```diff
@@ -0,0 +1,20 @@
+- win_video_play:
+    type = win_video_play
+    only Windows
+    image_snapshot = yes
+    x86_64:
+        program_files = "%ProgramFiles(x86)%"
+    i386:
+        program_files = "%ProgramFiles%"
+    #Disable first startup guide for windows media player
+    wmplayer_reg_cmd = "reg add HKLM\SOFTWARE\Policies\Microsoft\WindowsMediaPlayer /v GroupPrivacyAcceptance  /t REG_DWORD /f /d 00000001"
+    wmplayer_path = "${program_files}\Windows Media Player\wmplayer.exe"
+    #Install kmplayer if wmplayer is not installed default
+    kmplayer_install_cmd = "start /wait WIN_UTILS:\kmplayer\%s\KMPlayer-setup.exe /SP- /VERYSILENT"
+    kmplayer_path = "${program_files}\KMPlayer\kmplayer.exe"
+    #Disable IEHarden for kmplayer
+    kmplayer_reg_cmd = reg add "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap" /v IEHarden /t REG_DWORD /f /d 00000000 &
+    video_url = http://fileshare.com/Peppa_Pig_39_The_Tree_House.avi
+    play_video_cmd = '"%s" ${video_url} /play /fullscreen'
+    guest_alias = "Win2008-sp2-32:2k8\x86,Win2008-sp2-64:2k8\amd64,Win2008-r2-64:2k8\amd64,Win2012-64:2k12\amd64,Win2012-64r2:2k12\amd64"
```

Comment:
Win10 is Desktop, mplayer is installed default, and mplayer is also installed in Win2016